db/migrate/20170124170136_add_role_to_minions.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToMinions < ActiveRecord::Migration[5.0]
+  def change
+    add_column :minions, :role, :string, index: true, after: :hostname
```

Comment:
I think it makes more sense to have this as an integer and then in the model add an enum that maps that ([as we do in portus and team members](https://github.com/SUSE/Portus/blob/master/app/models/team_user.rb#L25)).